app/views/onboarding/OnboardingNotifications.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import PermissionsContext from '../../PermissionsContext';
+
+import { Icons, Images } from '../../assets';
+
+import OnboardingTemplate from './OnboardingTemplate';
+import { useTranslation } from 'react-i18next';
+import {
+  NavigationParams,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation';
+import { Screens } from '../../navigation';
+
+type OnboardingNotificationsProps = {
+  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
+};
+
+const OnboardingNotifications = ({
+  navigation,
+}: OnboardingNotificationsProps): JSX.Element => {
+  const { authSubscription, location } = useContext(PermissionsContext);
+
+  const requestNotifications = async () => {
+    await location.request();
```

Comment:
This should definitely be `notification` not `location`

Reply:
good catch yeah